Function Attributes/README.md

```diff
@@ -0,0 +1,560 @@
+# Function attributes
```

Comment:
Please place the `.md` files inside the `docs/` directory.
The GH Pages are built from the markdown files inside the `docs/` dir.
Please follow the structure defined in #23